### DISCUSSION TITLE
Bi-monthly Meeting

### DISCUSSION THREAD

**Discussion by brey** [oceanmodeling/Management#24] [Announcements]
# Calendar

@oceanmodeling/everyone Please find below the arrangement for the meeting management:

Our events are being tracked by a calendar with a [public url](https://calendar.google.com/calendar/u/0?cid=b2NlYW5tb2RlbGluZ2NvbGxhYm9yYXRpb25AZ21haWwuY29t).  **Feel free to subscribe to it.**

> Please note that you have to opt in for email notifications. See [here](https://support.google.com/calendar/answer/37242?hl=en&co=GENIE.Platform%3DDesktop&oco=1#zippy=%2Cchange-your-notification-settings%2Cturn-notifications-on-or-off).

To be able to post events yourself and/or integrate with your calendar app (e.g. Outlook), please send an `email` to  oceanmodelingcollaboration@gmail.com.

# Agendas and Minutes

Meeting minutes and presentations will be posted [here]( https://github.com/oceanmodeling/Management). 

# Purpose of the meetings 

This organization is conceived as a community. The bi-monthly meeting is its management function. So, we can shape it in the following ways:

- Propose actions to expand the community
- Suggest relevant research topics for discussions
- Present our work and/or our vision
- Invite others to present their work/vision
- Participate in the discussions

If interested to partake, please reach out to us in this space with proposals, early if possible, in order to shape the agenda in advance of the next meeting.

**Comment by tomsail** [comment]
## Subject: Reminder: Next Bimonthly Meeting – Thursday 6th, February at 1500 CET.

Dear @oceanmodeling/everyone,

The next bimonthly meeting is scheduled for next Thursday 6th, February at 1500 CET.

We’ve prepared a short draft [agenda](https://github.com/oceanmodeling/Management/blob/main/meetings/20250206.md) for the meeting, which currently includes:

1. Community Management
2. Simplifying Coastlines for Mesh Generation – Presented by @pmav99 (Panos, EC-JRC)
   - OSM Use Case

If you have any topics, presentations, or ideas you’d like to add to the agenda, please don’t hesitate to share them! Your input is highly valued and helps ensure our discussions are productive and inclusive.

Looking forward to seeing you all next Thursday

**Reply by tomsail** [reply]
Dear @oceanmodeling/everyone, 

Please connect to the meeting in 30min using this JITSI link: 
[outdated]
It should work in any browser

**Comment by tomsail** [comment]
Hi all, 

In order to anticipate the next meeting (3/4/2025), we'd like to secure some presentations and topics. 

@kiaghadi  already accepted to present his work and activities.

@ebrahimhamidi would you be keen to briefly present your work and activities too?

If anyone is interested to add an item to the agenda, please reach out directly here.

**Reply by tomsail** [reply]
# Reminder: Next Bimonthly Meeting – Thursday 3rd of April at 1500 CET.

Dear @oceanmodeling/everyone, 

The [agenda](https://github.com/oceanmodeling/Management/blob/main/meetings/20250403.md) for next Thurday's meeting has been updated. It will feature presentations from the speakers tagged above. 

If anyone wishes to see any other topic, presentations or ideas added to the agenda, please don’t hesitate to push suggestions here. This space is also a platform for broader discussions on the state of the art, best practices, collaborations and updates around ocean modeling.

Like the last times, we will share a JITSI meet link a few moments before the meeting. 

If you haven't done it already, add the [meeting event](https://calendar.google.com/calendar/u/0/r?cid=b2NlYW5tb2RlbGluZ2NvbGxhYm9yYXRpb25AZ21haWwuY29t) to your calendar.

Looking forward seeing you next Thursday

**Reply by tomsail** [reply]
Dear @oceanmodeling/everyone ,
 
Few announcements:
 
- I will clean up the comments on this discussion to reduce confusion and improve readability
- Meeting Notes (Cc @hga007): While we don't plan on posting recordings, I'll try to take meeting notes and share it afterhand on the [management](https://github.com/oceanmodeling/Management/blob/main/meetings/20250403.md) repository.

**Reply by tomsail** [reply]
Dear all, please find the meeting minutes of the last bimonthly meeting here: https://github.com/oceanmodeling/Management/tree/main/meetings

You'll find also the pdf of @kiaghadi2013 and @ebrahimhamidi presentations.

**Comment by tomsail** [comment]
## Reminder: Next Bimonthly Meeting – Thursday 5rd of June at 1500 CET.

Dear @oceanmodeling/everyone, 

Please find the [agenda](https://github.com/oceanmodeling/Management/blob/main/meetings/20250605.md) for the next bimonthly meeting, which currently includes:
1. Community management
2. Cohorts scheduling on HPC for typhoon modeling by @danishyo

link to connect to the meeting is the following: https://moderated.jitsi.net/01453785be8a4b2b83d5fb9013a603be6a6ae6ca0c9941c8a7746877dbcbca99

Looking forward seeing you all next Thursday

**Comment by tomsail** [comment]
## Cancelled: Next Bimonthly Meeting - Thursday 7th of August at 1500 CET.

Dear @oceanmodeling/everyone, 
Due to the holidays and personal agendas, the bimonthly meeting next week is cancelled.
We'll reconvene in 2 months for the next one, the first Thursday of October.

Wishing you all a great summer

**Comment by tomsail** [comment]
## Reminder: Next Bimonthly meeting - Thursday 2nd of October at 1500 CET (0900 EST)

Dear @oceanmodeling/everyone, 

Here is the agenda for our next bimonthly meeting, coming up this Thursday. 

 * Community Management
 * @cmbarton - Michael Barton - Director of the [Open Modeling Foundation](https://www.openmodelingfoundation.org/). Presentation of the OMF and return on the experience of building a global virtual community. 
 * @felicio93 - Felicio Cassalho - Tidal Modulation of Waves in the Aleutian Islands
 
## Meting Link
Please use the following link to connect to the meeting on JITSI: 
https://moderated.jitsi.net/87aca17a1b354a2d8cdb85c20e73e4de73824f42df36405d933e4b5c296e4f57
 
Looking forward seeing you all, 
 
PS:  To the speakers, just react with 👍🏼 to confirm you got the notification. 
PPS: For those that haven't lock the meeting on their calendar yet, please have a look at the [top post on this discussion](https://github.com/orgs/oceanmodeling/discussions/24#discussion-7657254)

**Reply by platipodium** [reply]
Just to make sure: we're still on summer time, so 15:00 CEST, correct?

**Reply by tomsail** [reply]
Yes @platipodium 
Meeting is at 15:00 CEST so 06:00 Arizona Time.
@cmbarton do you want to switch with Felicio to give you more 30min ?

**Reply by felicio93** [reply]
Hi @tomsail and @cmbarton. I am more than happy to present first. Also my presentation should not take more than 30 minutes.

**Comment by tomsail** [comment]
# Cancelled: Next Bimonthly Meeting - Thursday 4th of December at 1500 CET.

Dear @oceanmodeling/everyone, 

Due to scheduling conflicts, the next bimonthly meeting is unfortunately cancelled.
We’ll meet again in two months, on the first Thursday of February, for our next session.

Wishing you all a wonderful end-of-year holiday season

**Comment by tomsail** [comment]
# Next Bimonthly meeting - Thursday 5th of February at 1500 CET (0900 EST)

Dear @oceanmodeling/everyone,

I hope you all the started the year well, 

Next week's bimonthly meeting's theme is on ML & (sea level) time series. 

[Agenda](https://github.com/oceanmodeling/Management/blob/main/meetings/20260205.md)'s items are the following:
1. Introduction and word on oceanmodeling community
2. RTide: Automating the Tidal Response Method by @thomasmonahan (University of Oxford)
3. Using Transformers for flood modeling by Amitay Sicherman and @gauchm (Google Research)
4. [If time allows] [`ioc_cleanup`](https://tomsail.github.io/ioc_cleanup/): reproducible cleaning of tide gauges by EC-JRC

The link to connect to the meeting is the following (moderator link so everyone can share screen and notes): https://moderated.jitsi.net/87aca17a1b354a2d8cdb85c20e73e4de73824f42df36405d933e4b5c296e4f57

Looking forward to seeing you all next Thursday!

PS: if done already, add the [event](https://github.com/orgs/oceanmodeling/discussions/24#discussion-7657254) on your calendar.

**Reply by tomsail** [reply]
Anyone having difficulties to join, come here : 
https://meet.jit.si/moderated/3c414578d802320df70c78b6e1d154f30cd4ca759137ce25d11832401a31523a

**Reply by tomsail** [reply]
Thanks @oceanmodeling/everyone for the participation and for the speakers. 
I will add in the coming days the transcript of the meeting and the presentations (if speakers agree to share them)

**Comment by saeed-moghimi-noaa** [comment]
@tomsail 
Thanks you and the presenters for today very lively and interesting meeting.

**Comment by tomsail** [comment]
Dear @oceanmodeling/everyone, 

Unfortunately due to a lack of presenters and the Easter holidays, we'll cancel the next Bimonthly meeting. 
For the next one in June, we'll make sure to confirm more speakers and engage with you all to push more interesting topics every time. 

Best

**Comment by tomsail** [comment]
# Next Bimonthly Meeting - Thursday 4th of June 1500 CET (0900 EST)

Dear @oceanmodeling/everyone , 

We've been struggling to secure speakers and presentations for the last meetings. 

We'll nonetheless maintain our next meeting, and discuss on alternatives for meeting formats and ways to exchange & collaborate on ocean modeling best practices. 

Looking forward to seeing you all next Thursday !

**Reply by tomsail** [reply]
Dear all, please find the link to connect in an hour: 

https://meet.jit.si/moderated/3c414578d802320df70c78b6e1d154f30cd4ca759137ce25d11832401a31523a

**Comment by tomsail** [comment]
# Cancelled: Next Bimonthly Meeting - Thursday 6th of August at 1500 CET.

Dear @oceanmodeling/everyone,

As anticipated in the last meeting, the next bimonthly meeting is cancelled.
We’ll reconvene in two months, on the first Thursday of October, for our next session.

I wish you all a nice holiday break

**Comment by tomsail** [comment]
# Next Bimonthly Meeting - Thursday 1st of October 1500 CET (0900 EST)

Dear @oceanmodeling/everyone,

Here is the agenda for next week's meeting:

1. Community Management
2. Short presentations (5min) on new/updated repositories
   - [`oceanmcp`](https://github.com/oceanmodeling/ocean-mcp) by @mansurjisan
   - [`detide`](https://github.com/oceanmodeling/detide) by @tomsail
   - [`coastal-surge-processing`](https://github.com/oceanmodeling/coastal-surge-processing) by @WPringle
   - [`ioc_cleanup`](https://github.com/oceanmodeling/ioc_cleanup) by @tomsail
3. Main presentation: Automatic Quality Analysis & Quality Control (QA/QC) of tide gauges using AI
   - overview and plans for integration of the AI model by @LindsayRAbrams
   - AI model framework by @ltdung
   - [if time allows] results on filling gaps in historic data by Dr. Phillipe Tissot
   - Discussion

We will try to keep repositories presentations short (5min) in order to leave enough time for the main presentation and discussion. 

Link to connect is the following: 
https://meet.jit.si/moderated/a443922b34f70c5b73486e096b027713f968506ed2daaf4f0725840a99b933dd

Looking forward to see you all next Thursday!
